tools/buildgen/plugins/expand_filegroups.py

```diff
@@ -47,11 +47,10 @@ def uniquify(lst):
 def mako_plugin(dictionary):
     """The exported plugin code for expand_filegroups.
 
-  The list of libs in the build.yaml file can contain "filegroups" tags.
-  These refer to the filegroups in the root object. We will expand and
-  merge filegroups on the src, headers and public_headers properties.
-
-  """
+    The list of libs in the build.yaml file can contain "filegroups" tags.
```

Comment:
btw, I think since the migration to extract_metadata_from_bazel.py, we might no longer have any filegroups - it might be worth considering getting rid of this "plugin".

Reply:
After removing the `tools/buildgen/plugins/expand_filegroups.py` script, plugin `tools/buildgen/plugins/make_fuzzer_tests.py` failed.

Reply:
sg.